### PR TITLE
Finalize CI

### LINKED ISSUE #41: specimen: days_to_collection field
This should be mapped to the **time_of_collection** field of the [GA4GH Biosample](https://phenopacket-schema.readthedocs.io/en/latest/biosample.html) message.

Check what data is coming for this. Probably use the same days to ISO 8601 function as for disease and individual.

### LINKED ISSUE #62: Getting continuous integration working in github actions

### PR DESCRIPTION
**Pull request by ielis**
The PR works on finalizing tests and re-enabling CI and addresses several points.

# Major

## Mapping CDA diagnosis to NCIT neoplasm term

@pnrobinson @justaddcoffee the disease term mapping is $\pm$ the same as was previously. I, however, rearranged the tests to only test at one site. Your tests did not disappear, but I removed redundancies and moved them to just one place.

We still need to work on the mapping details, e.g. by updating the `cda_to_ncit_map.tsv` file, but the mapping should work most of the time.

A question for @pnrobinson - there are 2 mapping files that seem to achieve the same: [cda_to_ncit_map.tsv](https://github.com/monarch-initiative/oncoexporter/blob/minor-tweaks/src/oncoexporter/ncit_mapping_files/cda_to_ncit_map.tsv) and [cda_to_ncit_map_old.tsv](https://github.com/monarch-initiative/oncoexporter/blob/minor-tweaks/src/oncoexporter/ncit_mapping_files/cda_to_ncit_map_old.tsv) (used to be at `oncoexporter/cda/mapper/neoplasm_types.tsv`). However, the latter seems to be more complete. @pnrobinson is there a difference between the files?

## Ensure the driver scripts work

The driver scripts for ETL of Lung and Cervix cohorts should now work. @pnrobinson @justaddcoffee please verify on your end.

# Minor

## Mapping of days of age to ISO8601 duration

Next, We map the number of days of life into ISO8601 consisting of days only (e.g. `11000 -> 'P11000D'`) instead of rounding the days to years, months, etc. This is done in order to preserve the data and we can add an age formatter later, if needs be.
The days of life mapping also relates to #70, #71 .

## Use pytest for new tests, keep existing `unittest`

We prefer using `pytest` as the test runner and all new tests should use Pytest. Pytest can, however, handle `unittest` tests. Therefore, there is no need to upgrade the tests right now.

Fixes #41, #62, #66

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # oncoexporter
+
```
Comment (justaddcoffee) [inline README.md:2]: +1!

File: src/oncoexporter/cda/mapper/op_uberon_mapper.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+from typing import Optional
 
 from .op_mapper import OpMapper
-from typing import Optional
 import pandas as pd
-from collections import defaultdict
 import phenopackets as PPkt
 
+
 class OpUberonMapper(OpMapper):
```
Comment (justaddcoffee) [inline src/oncoexporter/cda/mapper/op_uberon_mapper.py:8]: Eventually we should rename these "Op*" variables and class names. Op I think used to mean oncopackets, but now is just confusing

**Review by justaddcoffee — APPROVED** [review]

File: src/oncoexporter/cda/mapper/op_uberon_mapper.py (hunk 2 above)
Comment (ielis) [reply src/oncoexporter/cda/mapper/op_uberon_mapper.py:8]: Yes, it is a bit confusing. We should rename when we're ready for the change.

Comment (justaddcoffee) [reply src/oncoexporter/cda/mapper/op_uberon_mapper.py:8]: obviously this is not in scope for this PR, just thought I would mention this while I was reviewing the code

File: README.md (hunk 1 above)
Comment (justaddcoffee) [reply README.md:2]: +1 here was referring to the CI badge, which is pretty slick

**Pull request merged**